jetcd-discovery-dns-srv/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not sure if main jetcd repo should have dns resolver implementation. the users can write one themself.
I think its better to keep code base simple and small.

Reply:
Well, it was already included before so I do not see the point to remove it and I think we should provide reasonable default implementation for common things like this one.

Reply:
I see.